File: sgdk/new_screens/record_screen.c
#include "record_screen.h"
#include "../engine/enum_manager.h"
#include "../engine/font_manager.h"

void screen_record_screen_load()
{
	engine_font_manager_text( "RECORD SCREEN!!", 10, 2 );
}

void screen_record_screen_update( unsigned char *screen_type )
{
	*screen_type = screen_type_record;
}

File: TurtleBridge/physics/platformer/03-testCommandMgr/Test/Common/Screens/RecordScreen.cs
using Microsoft.Xna.Framework;
using WindowsGame.Master;

namespace WindowsGame.Common
{
	public class RecordScreen : IScreen
	{
		public void Initialize()
		{
		}

		public void LoadContent()
		{
			Engine.Game.Window.Title = GetType().Name;
		}

		public int Update(GameTime gameTime)
		{
			return (int)ScreenType.Record;
		}

		public void Draw()
		{
			Engine.SpriteBatch.DrawString(Assets.EmulogicFont, "RECORD SCREEN", new Vector2(10, 10), Color.Yellow);
		}
	}
}
